packages/labs/analyzer/src/lib/javascript/modules.ts

```diff
@@ -135,7 +140,7 @@ export const getModule = (
         if (moduleSpecifier === undefined) {
           throw new DiagnosticsError(
             statement,
-            `Expected a wildcard export to have a module specifier.`
+            `Internal Error: expected a wildcard export to always have a module specifier.`
```

Comment:
Diagnostic vs internal.

If we have these cases where we don't think valid source should be reach here, then I guess we'd expect a semantic error to already have been produced by TypeScript? Can we just assume that and carry on, or do we really want to halt execution?